packages/server/src/ui/visual/HeroBar/index.tsx

```diff
@@ -3,11 +3,17 @@ import InstallIcon from './install-icon.svg';
 // tslint:disable-next-line:no-implicit-dependencies
 import background from '!url-loader!./background-image.svg';
 
-export function InstallButton({size = 'sm'}: {size?: 'sm' | 'lg'}) {
+export function InstallButton({
+  size = 'sm',
+  shadow = 'gray',
+}: {
+  size?: 'sm' | 'lg';
+  shadow?: 'gray' | 'orange' | 'white';
+}) {
```

Comment:
Constructing class names like this will break in production. Just accept a `className?: string` prop and add that into the class list.

The issue is that there's a production optimisation that checks each class name in the CSS to see if it's used in code, and removes the un-used css classes. It's not super smart, so it's just looking for occurrences of that string in the JavaScript code that's output, so it wont' be able to connect `focus:shadow-${shadow}` with the fact that you pass in `shadow` somewhere else.

Reply:
updated, but I've gone with setting shadow options to 'shadow-gray' rather than just 'gray' etc.  This is because I want to default the shadow to 'shadow-gray' and className doesn't feel like it should just be defaulted if empty.  Is this ok?